content/en/docs/concepts/services-networking/topology-aware-routing.md

```diff
@@ -1,11 +1,11 @@
 ---
 reviewers:
 - robscott
-title: Topology Aware Hints
+title: Topology Aware Routing
```

Comment:
How will we avoid confusion between this and https://k8s.io/docs/concepts/services-networking/service-topology/?

Reply:
Can we safely remove the previous doc since it's been removed for 6 k8s versions/2 years and only ever made it to alpha?